modules/highgui/CMakeLists.txt

```diff
@@ -29,6 +29,10 @@ file(GLOB highgui_ext_hdrs
      "${CMAKE_CURRENT_LIST_DIR}/include/opencv2/${name}/*.hpp"
      "${CMAKE_CURRENT_LIST_DIR}/include/opencv2/${name}/*.h")
 
+# Removing WinRT API headers by default
+status("  ${name}: Removing WinRT API headers by default")
```

Comment:
Remove this message.
It is too verbose on Linux, Android, iOS builds and actually on Windows. Try to use minimal set of messages.